### PR TITLE
Add sequence composition visibility options on save - PMT #109928

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: mediathread/templates/projects/sequence.html [hunk 1]
```diff
@@ -127,12 +167,10 @@
                 <div id="jux-container"></div>
             </div>
             <div role="tabpanel" class="tab-pane fade" id="reflection">
-                {% if project.is_submitted %}
-                    {{project.body|safe}}
-                {% else %}{% if response_can_edit %}
+                {% if response_can_edit %}
```
Comment (sdreher) [inline mediathread/templates/projects/sequence.html]: The project body should be visible (but not necessarily editable) to anyone who is viewing the response.

File: mediathread/templates/projects/sequence.html [hunk 2]
```diff
@@ -71,6 +71,46 @@
...
+                                            <input checked="checked" id="id_publish_0" name="publish" type="radio" value="PrivateEditorsAreOwners"> Draft - only you can view
+                                        </label>
+                                    </li>
+                                    <li>
+                                        <label for="id_publish_1">
+                                            <input id="id_publish_1" name="publish" type="radio" value="CourseProtected"> Whole Class - all class members can view
+                                        </label>
+                                    </li>
+                                    <li>
+                                        <label for="id_publish_2">
```
Comment (sdreher) [inline mediathread/templates/projects/sequence.html]: Publish to World should be hidden/shown based on the course settings

File: mediathread/templates/projects/sequence.html (hunk 1 above)
Comment (nikolas) [inline mediathread/templates/projects/sequence.html]: Done

Comment (nikolas) [inline mediathread/templates/projects/sequence.html]: Done

**Pull request merged**